tests/test_pytask_io.py

```diff
@@ -151,12 +147,12 @@ def send_email_quick(msg):
         assert result["serialized_result"] == "Hello Joe 1"
 
     def test_add_unit_of_work(self):
-        
+
         meta = self.pytask.add_task(send_email, "Hello world!")
         # TODO: Fix this, `brpop` is blocking as key `tasks` is not present.
         # assert serialize_unit_of_work(send_email, "Hello world!") in self.r.brpop("tasks")
 
         # Test if the uow is on the queue
         # Test if the uow is on the store
 
-        # assert asyncio.get_running_loop() == True
+        # assert asyncio.get_running_loop() == True
```

Comment:
These tests need updating but also can be merged with @MakuZo test work once thats reopened

Reply:
@joegasewicz It seems that I cannot reopen my PR, but that may be because I'm on my phone right now.

Also offtopic, is there any channel we can communicate by? Because Gitter seems closed.

Reply:
no problem, I'll re create another gitter channel. I will wait for your PR to re open because I want to make sure we have all the tests from your branch first in master before i merge this in, thanks @MakuZo

Reply:
ok np, ill merge this in now without tests and we can up the tests with your pr later
thanks